guides/plugins/apps/content/cms/add-custom-cms-blocks.md

```diff
@@ -128,13 +138,17 @@ In our example we will define two custom CMS blocks:
 {% endcode %}
 
 Let's have a look at how to configure a CMS block from your app's `cms.xml`:
-* `name` : A **unique** technical name for your block. This name is also used to discover the block's `preview.html` and `styles.css` in the `Resources/cms/blocks/` directory.
-* `category` : Blocks are divided into categories which can be found in [src/Administration/Resources/app/administration/src/module/sw-cms/blocks](https://github.com/shopware/platform/tree/trunk/src/Administration/Resources/app/administration/src/module/sw-cms/blocks)
-* `label` : The **translatable** label will be shown in the Administration.
-* `default-config` : Some default configuration for the block.
-* `slots` : Each block holds [slots](https://vuejs.org/v2/guide/components-slots.html) that configure which elements they show.
+`name` : A **unique** technical name for your block. This name is also used to discover the block's `preview.html` and `styles.css` in the `Resources/cms/blocks/` directory.
 
-Here you can find the complete [CMS Manifest](../../../../../resources/references/app-reference/cms-reference.md).
+`category` : Blocks are divided into categories which can be found in [src/Administration/Resources/app/administration/src/module/sw-cms/blocks](https://github.com/shopware/platform/tree/trunk/src/Administration/Resources/app/administration/src/module/sw-cms/blocks)
+
+`label` : The **translatable** label will be shown in the Administration.
+
+`default-config` : Some default configuration for the block.
+
+`slots` : Each block holds [slots](https://vuejs.org/v2/guide/components-slots.html) that configure which elements they show.
```

Comment:
we can't just link the documentation for Vuejs slots here. The storefront doesn't use Vuejs. Maybe we should change this and tell the user that it's similar to the slot system of Vuejs but with less functionality.

Reply:
Ah okay, I just adopted that from the [plugin guide](https://developer.shopware.com/docs/guides/plugins/plugins/content/cms/add-cms-block#rendering-the-block), but I'll remove it here.